CHANGES

```diff
@@ -21,6 +21,10 @@
      (CVE-2019-1551)
      [Andy Polyakov]
 
+  *) Most memory-debug features have been deprecated, and the function
```

Comment:
Shouldn't _function_ be plural or _no-ops_ singular?

Reply:
Good catch.  I went with "the functionality..."